packages/cli/commands/sandbox/create.js

```diff
@@ -146,11 +149,14 @@ exports.handler = async options => {
       }),
     });
   } catch (err) {
+    debugErrorAndContext(err);
```

Comment:
Ended up using `debugErrorAndContext` instead of the full `logErrorInstance` as the latter was providing too much detail into the specific error here. 

For example, the first log is with `logErrorInstance` and the second is maintaining the old log that I have except with the extra `debugErrorAndContext`:

<img width="1190" alt="Screen Shot 2022-08-04 at 10 46 02 AM" src="https://user-images.githubusercontent.com/16788677/182876610-ee72316c-017e-401f-b094-7be2b3898b05.png">

<img width="1190" alt="Screen Shot 2022-08-04 at 10 45 27 AM" src="https://user-images.githubusercontent.com/16788677/182876663-594cd3fa-04e6-4a7d-b144-4b1382bd41f6.png">